src/OffscreenContextGLX.cc

```diff
@@ -256,11 +256,25 @@ bool create_glx_dummy_context(OffscreenContext &ctx)
 	int major;
 	int minor;
 	auto result = false;
-
-	ctx.xdisplay = XOpenDisplay(nullptr);
-	if (ctx.xdisplay == nullptr) {
+	int tries = 0;
+	auto dpyenv = getenv("DISPLAY");
+	auto retryenv = getenv("OPENSCAD_X_CONN_RETRY");
+	int connect_retry_count = retryenv ? atoi(retryenv) : 0;
+
+	for (;;) {
+		ctx.xdisplay = XOpenDisplay(nullptr);
+		tries++;
+		if (ctx.xdisplay != nullptr) {
+			break;
+		}
+		if (tries < connect_retry_count) {
```

Comment:
Shouldn't that be <= to actually have 3 retries with an env setting of 3? I wonder if we should clamp the value to a maximum of 10 or so?